account/userprofile/service.go

```diff
@@ -0,0 +1,72 @@
+package userprofile
```

Comment:
just call it userinfo? because we already have keycloak user profile, would start getting confusing.

Reply:
or just user?

Reply:
might be confusing because we have a user.go already :)

Reply:
userprofile package is used by both user.go and userinfo.go

Reply:
makes sense. Let's go with `user` as the package name. And please address review comments using the commit IDs wherever possible. :)

Reply:
sure